scripts/steam.js

```diff
@@ -1,34 +1,25 @@
-/* global require __dirname */
-const fs = require('fs')
+/* global require  module */
 const delay = require('delay')
-const {resolve} = require('path')
-const {steam, getTopRatedGames} = require('./lib')
+const {steam, load, saveToJson} = require('./lib')
 
-async function main() {
-  let games = []
-
-  console.log('Get top rated games from steamdb...')
-  try {
-    games = await getTopRatedGames()
-  } catch (e) {
-    console.log('Retrieval of top rated games from steamdb.info failed', e)
-  }
-
-  const gamesWithDetails = []
+const steamGames = load('steam-games')
 
-  console.log(`Retrieved list with ${games.length} games from steamdb\n`)
+async function main() {
+  const games = load('top-games-steamdb')
```

Comment:
naming "steamGames", "games" is confusing

Maybe go for steamAppDetails and steamDbGames